templates/registration/profile.html

```diff
@@ -1,50 +1,75 @@
+                <li class="tabs-title"><a href="#profile-stats"><i class="fa fa-bar-chart fa-2x tab-icon" aria-hidden="true"></i><span class="profile-name">Earnings</span></a></li>
+                <li class="tabs-title"><a href="#portfolio"><i class="fa fa-user fa-2x tab-icon" aria-hidden="true"></i><span class="profile-name">Artists</span></a></li>
+                <li class="tabs-title"><a href="#music"><i class="fa fa-music fa-2x tab-icon" aria-hidden="true"></i><span class="profile-name">Music</span></a></li>
             </ul>
 
             <div class="tabs-content" data-tabs-content="profile-tabs">
-                <div class="tabs-panel is-active" id="portfolio">
+                <div class="tabs-panel is-active" id="updates">
+                    <h2>ACTIVITY</h2>
+                    {% if updates %}
+                        {% include "artist/includes/artist_updates.html" with updates=updates show_artist_name=True %}
+                    {% else %}
+                        <p>This is where you will recieve updates from your artists.</p>
+                        <p>Looks like they haven't posted yet, but when they do it'll feed right here</p>
+                    {% endif %}
+                </div>
+                <div class="tabs-panel" id="profile-stats">
+                    {% include "registration/includes/profile_stats.html" %}
+                </div>
+                <div class="tabs-panel" id="portfolio">
                     {% include "registration/includes/profile_portfolio.html" %}
                 </div>
-                {% if albums %}
-                    <div class="tabs-panel" id="music">
-                        {% include "registration/includes/profile_music.html" %}
-                    </div>
-                {% endif %}
-                {% if updates %}
-                    <div class="tabs-panel" id="updates">
-                        <h2>ARTIST UPDATES</h2>
-                        {% include "artist/includes/artist_updates.html" with updates=updates show_artist_name=True %}
-                    </div>
-                {% endif %}
+                <div class="tabs-panel" id="music">
+                    {% include "registration/includes/profile_music.html" %}
+                </div>
             </div>
         {% else %}
             {% include "registration/includes/profile_portfolio.html" %}
         {% endif %}
     {% else %}
         <p>You haven't invested in anything yet!</p>
         <div class="profile-boxes">
-            <a class="button text-center box-margin view-artists-box" href="{% url 'artists' %}?campaign-status=Active">START INVESTING</a>
+            <a class="button text-center box-margin view-artists-box" href="{% url 'artists' %}">START INVESTING</a>
         </div>
     {% endif %}
 {% endblock %}
+
+{% block extrajs %}
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.1.4/Chart.min.js"></script>
```

Comment:
It's an outdated version and doesn't work with the new charts. I can updated it or add the new one of that works better